sqlite_utils/db.py

```diff
@@ -3011,6 +3048,85 @@ def truncate(value):
             least_common,
         )
 
+    def add_geometry_column(
+        self,
+        column_name: str,
+        geometry_type: str,
+        srid: int = 4326,
+        coord_dimension: str = "XY",
+        not_null: bool = False,
+    ) -> bool:
+        """
+        In Spatialite, a geometry column can only be added to an existing table.
+        To do so, use ``table.add_geometry_column``, passing in a geometry type.
+
+        By default, this will add a nullable column called ``geometry`` using
```

Comment:
The docs here say that the default will be a "geometry" but line 3054 doesn't yet set that default.

Reply:
Fixed.